controllers/registerController.js

```diff
@@ -42,7 +48,7 @@ exports.postRegistraionInfo = (req, res, next) => {
                         playerID: result.insertedId
                 })
                 // need to return something json because frontend expects to receive `json.
-                .send({})
+                .send() // no deed empty {}, 
```

Comment:
Do we even need `send`? `json` functions probably does "send". we may not need `send` when using `json()`.